bokeh/tests/test_resources.py

```diff
@@ -19,7 +19,41 @@
 
 LOG_LEVELS = ['trace', 'debug', 'info', 'warn', 'error', 'fatal']
 
-DEFAULT_JOG_JS_RAW = 'Bokeh.set_log_level("info");'
+DEFAULT_LOG_JS_RAW = 'Bokeh.set_log_level("info");'
+
+
+class TestJSResources(unittest.TestCase):
+
+    def test_basic(self):
+        r = resources.JSResources()
+        self.assertEqual(r.mode, "inline")
+
+    def test_inline_has_no_css(self):
+        r = resources.JSResources(mode="inline")
+        self.assertEqual(r.mode, "inline")
```

Comment:
It's the `test_basic` that'll conflict, right? I would remove the test, but I actually don't think we've got test coverage for the low-level default anywhere. So I think it's best to keep it in and change it to the new default.